sources/system/file-system/unix-interface.dylan

```diff
@@ -106,6 +123,32 @@ define function unix-delete-file (path :: <byte-string>) => (ok :: <boolean>)
   end = 0;
 end function unix-delete-file;
 
+define function unix-copy-file
+    (source-fd :: <integer>, destination-fd :: <integer>, #key block-size = 2097152)
+ => ()
+  let buffer = make(<buffer>, size: block-size);
+  let address 
+    = primitive-wrap-machine-word
+        (primitive-cast-pointer-as-raw
+          (primitive-string-as-raw(buffer)));
+
+  block (finished?)
```

Comment:
It's a little unusual for the exit-block function to be named as a predicate (with "?").  I'd just call it "finished" or "done".  There's no need to pass any arguments (`#t`) to it since unix-copy-file returns no values.

Reply:
Done in next comment.